CMakeLists.txt

```diff
@@ -900,9 +900,9 @@ include(ExternalProject)
 
 # ---[ Dependencies ---[ FBGEMM doesn't work on x86 32bit and
 # CMAKE_SYSTEM_PROCESSOR thinks its 64bit
-if(USE_FBGEMM AND NOT CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+if(USE_FBGEMM AND NOT CMAKE_SYSTEM_PROCESSOR MATCHES "^(x86_64|aarch64)$")
   message(WARNING
-    "x64 operating system is required for FBGEMM. "
+    "64-bit operating system is required for FBGEMM. "
```

Comment:
This is a wrong statement. This  will not work on RISCV or S390, will it? If one really wants to check for 64-bit OS, they need to modify the condition to `CMAKE_SIZEOF_VOID_P EQUALS 8`

Reply:
Looks good to me, thank you for the update